forwards_paper.tex

```diff
@@ -1158,11 +1211,12 @@ \section{Simplification algorithm}
     }
 
 \end{taocpalg}
+\krt{I think the algo discussion is  clear.  I've not tried to implement it, which would be the real test, but I find the description fine other than needing to define $Q$ when first used.}
 
-\newpage
-\clearpage
-\section*{Supplementary Information}
-\renewcommand{\thefigure}{S\arabic{figure}}
+%% \newpage
+%% \clearpage
+%% \section*{Supplementary Information}
+\renewcommand{\thefigure}{A\arabic{figure}}
```

Comment:
Adding something like `\section{Further simulation details}`, changing `A` to `B` and inserting paragraphs from above would make this a more stand-alone appendix